plugins/CoreAdminHome/CustomLogo.php

```diff
@@ -196,18 +196,22 @@ private function uploadImage($uploadFieldName, $targetHeight, $userPath)
         list($width, $height) = getimagesize($file);
         switch ($_FILES[$uploadFieldName]['type']) {
             case 'image/jpeg':
-                $image = imagecreatefromjpeg($file);
+                $image = @imagecreatefromjpeg($file);
```

Comment:
Cool, thx. Can you have a look at the other comment? Not sure if something is do on the other comment. If not, feel free to merge